src/game/GameProcMain.cpp

```diff
@@ -1029,6 +1029,9 @@ void CGameProcMain::ProcessLocalInput(DWORD dwMouseFlags) {
     if (dwMouseFlags & MOUSE_LBDBLCLK) {
         OnMouseLDBtnPress(ptCur, ptPrev);
     }
+    if (dwMouseFlags & MOUSE_MBCLICK) {
+        OnMouseMBtnPress(ptCur, ptPrev);
+    }
```

Comment:
Unfortunately that's not how it works. Here it suppose to set the rotation value to 0.0f that will be used in `CGameProcMain::Tick`:
```suggestion
    if (dwMouseFlags & MOUSE_MBCLICK) {
        m_fRotateValue = 0.0f;
    }
```